### PR TITLE
Add favorite weather locations

### PR DESCRIPTION
**Pull request by julien-nc**
Basic weather location favorites inside the weather status widget.

ccc

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]
This is getting super fancy!!
Code looks good! :ship:

File: apps/weather_status/src/App.vue [hunk 1]
```diff
@@ -26,13 +26,19 @@
...
+				<ActionLink v-if="gotWeather"
 					icon="icon-address"
 					target="_blank"
 					:href="weatherLinkTarget"
 					:close-after-click="true">
 					{{ locationText }}
 				</ActionLink>
+				<ActionButton v-if="gotWeather"
+					:icon="addRemoveFavoriteIcon"
+					:close-after-click="false"
```
Comment (skjnldsv) [inline apps/weather_status/src/App.vue:38]: Btw, closeAfterClick is false by default
https://nextcloud-vue-components.netlify.app/#/Components/Actions?id=actionbutton

Comment (eneiluj) [reply apps/weather_status/src/App.vue]: Right, thanks.

**Review by GretaD — APPROVED** [review]

**Comment by eneiluj** [comment]
@rullzer @GretaD @skjnldsv What do you think about backporting this in stable20?

**Pull request merged**